package.nls.json

```
@@ -419,6 +420,8 @@
     "AWS.s3.downloadFile.progressTitle": "Downloading {0}...",
     "AWS.s3.downloadFile.saveButton": "Download",
     "AWS.s3.downloadFile.error.general": "Failed to download file {0}",
+    "AWS.s3.promptTime.placeHolder": "Defaults to 15 minutes",
+    "AWS.s3.promptTime.prompt": "Please enter the valid time (in minutes) for ${0} URL",
```

"Link duration (in minutes)"

---

Avoid "please" in messages.

```suggestion
    "AWS.s3.promptTime.prompt": "Enter the valid time (in minutes) for ${0} URL",
```